apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-preview-dropdown-selected.js

```diff
@@ -0,0 +1,33 @@
+import { __ } from '@wordpress/i18n';
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_template_part_detach_blocks`.
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export default () => ( {
+	id: 'wpcom-preview-dropdown-selected',
+	selector: `.components-dropdown-menu__menu[aria-label="${ __(
+		'View options'
+	) }"] .components-menu-item__button`,
+	type: 'click',
+	capture: true,
+	handler: ( _event, target ) => {
+		const mapTextToPreviewMode = {
+			[ __( 'Desktop' ) ]: 'desktop',
+			[ __( 'Desktop (50%)' ) ]: 'zoom-out',
+			[ __( 'Tablet' ) ]: 'tablet',
+			[ __( 'Mobile' ) ]: 'mobile',
+			[ `${ __( 'View site' ) }\n${ __( '(opens in a new tab)' ) }` ]: 'view-site',
+			[ __( 'Preview in new tab' ) ]: 'preview',
```

Comment:
Seems this one doesn't work because "Preview in new tab" is not inside a `.components-menu-item__button` element:

<img width="794" alt="Screenshot 2024-08-29 at 11 53 44" src="https://github.com/user-attachments/assets/a6eca459-fa19-45db-a922-97611b2d5662">

Reply:
Fixed 👍